src/components/AVPlayer/AVPlayer.js

```diff
@@ -359,12 +359,16 @@ class AVPlayer extends Component {
     }
   };
 
-  buildAppendData = (item, src, media) => {
+  buildAppendData = (item, src, props) => {
     return {
       unit_uid: item.unit.id,
       file_src: src,
-      current_time: media.currentTime,
-      duration: media.duration,
+      current_time: props.media.currentTime,
```

Comment:
If you need media, then call this function with props.media. And inside this function use destructuring.

Reply:
Taking all params now from this.